lib/publiccloud/instance.pm

```diff
@@ -352,40 +352,65 @@ sub wait_for_ssh {
     my $start_time = time();
     my $check_port = 1;
     my $sleep_period = $args{ignore_wrong_pubkey} ? 20 : 1;
-
+    #
+    my @sem = (1, 1, 1);
+    my @out = ('') x 3;
+    my @command = ('sudo systemctl is-system-running --wait', 'systemd-analyze time', 'sudo journalctl -b | grep -E "Reached target (Cloud-init|Default|Main User Target)"');
```

Comment:
Isn't one command enough?

Reply:
Yes, indeed infact as I told above, this PR is to verify _which_ command is the best one to use for this system-up definition. I put VRs results in the same test in order to compare them. So this PR is not to be merged as it is now, but at the end of evaluation, only one will be used.